pkgs/development/libraries/proj/proj-data.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+
+  # By default, no grids packages are installed due to a large size of this
+  # repository. Current size of all proj-data grids is almost 1GB and will only
+  # grow over the time.
+, gridPackages ? [ ]
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "proj-data";
+  version = "1.16.0";
+
+  src = fetchFromGitHub {
+    owner = "OSGeo";
+    repo = "PROJ-data";
+    rev = finalAttrs.version;
+    hash = "sha256-/EgDeWy2+KdcI4DLsRfWi5OWcGwO3AieEJQ5Zh+wdYE=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+    shopt -s extglob
+
+    mkdir -p $out
+    cp README.DATA $out/README-PROJ-DATA.md
+
+    for grid in ${builtins.toString gridPackages}; do
+      if [ ! -d $grid ]; then
+        echo "ERROR: proj-data grid $grid does not exist." >&2
+        exit 1
+      fi
+
+      cp $grid/!(*.sh|*.py) $out/
+    done
+
+    shopt -u extglob
+    runHook postInstall
+  '';
```

Comment:
Would you mind running this through `shellcheck`? It does have a handful of issues.